mpu6050/python/mpu6050Python/docker-compose.yml

```diff
@@ -5,6 +5,10 @@ services:
       context: .
       dockerfile: Dockerfile.debug
     image: ${LOCAL_REGISTRY}:5002/mpu6050python-debug:${TAG}
+    volumes:
```

Comment:
Should it be different for debug and production? (For debug it is volume here and for production a device?)

Reply:
In this case, I wanted to show that there're two possible ways to access I2C devices from within the container. When debugging, mounting the entire `/dev` directory and giving access to I2Cs using **cgroup rules** might make testing easier if errors occur. However, for production, it's not a good practice to give the container access to all I2C devices. That is covered by our peripheral article, but, if you prefer, I can use the same method for both debugging and production.

Reply:
Ideally one option will be chosen. However, if it makes sense to have one for development and one for production it is ok to have it, as long as there is documentation somewhere explaining the reason for it.